--- a/usr/share/applications/Programa2Mp.desktop
+++ b/usr/share/applications/Programa2Mp.desktop
@@ -1,15 +1,14 @@
 [Desktop Entry]
-Actions=Apkg-Verify;Apkg-Remove;
-Categories=Math;Science;Graphics;X-Red-Hat-Base;
+Categories=Math;Science;Education;
 Comment[en_US]=Satellite Image educational software for students between 8 and 16 years
-comment[es]=Software educativo con imagenes satelitales para jóvenes entre 8 y 16 años de edad
-Comment=Software educativo con imagenes satelitales para jóvenes entre 8 y 16 años de edad
+comment[es]=Software educativo con imágenes satelitales para jóvenes entre 8 y 16 años de edad
+Comment=Software educativo con imágenes satelitales para jóvenes entre 8 y 16 años de edad
 Encoding=UTF-8
 Exec=/usr/bin/programa2mp.sh %f
 FilePattern=
 GenericName[en_US]=Satellite Image educational software for students between 8 and 16 years
-GenericName[es]=Software educativo con imagenes satelitales para jóvenes entre 8 y 16 años de edad
-GenericName=Software educativo con imagenes satelitales para jóvenes entre 8 y 16 años de edad
+GenericName[es]=Software educativo con imágenes satelitales para jóvenes entre 8 y 16 años de edad
+GenericName=Software educativo con imágenes satelitales para jóvenes entre 8 y 16 años de edad
 Icon=programa2mp.png
 MimeType=application/pat
 Name[en_US]=Programa 2Mp
@@ -19,31 +18,5 @@
 StartupNotify=false
 Terminal=false
 TerminalOptions=
-TryExec=/usr/bin/Programa2Mp
-Type=Application
 Type=Application
 Version=1.2.4
-X-Autopackage=@suremptec.com/programa2mp:1.2.1
- 
-[Desktop Action Apkg-Verify]
-Exec=package verify programa2mp
-Icon=package
-Name[de]=Validieren Paket
-Name[en]=Verify Package
-Name[es]=Verifique El Paquete
-Name[fr]=Vérifier l'emballage
-Name[it]=Verifichi Il Pacchetto
-Name[nl]=Pakket Controleren
-Name=Verify Package
- 
-[Desktop Action Apkg-Remove]
-Exec=/usr/libexec/autopackage/autosu-gtk --root-only package remove programa2mp
-Icon=package
-Name[de]=Entfernen Paket
-Name[en]=Remove Package
-Name[es]=Quite El Paquete
-Name[fr]=Enlevez Le Paquet
-Name[it]=Rimuova Il Pacchetto
-Name[nl]=Pakket Verwijderen
-Name=Remove Package
- 
